### DISCUSSION TITLE
登陆 注册 密码重置页面的输入框消失

### DISCUSSION THREAD

**Discussion by SRY-CTB** [bs-community/blessing-skin-server#556] [Q&A]
### Blessing Skin 版本

6.0.2

### PHP 版本

8.1

### 出现问题时所使用的浏览器

Chrome, Microsoft Edge

### 您正在使用的 Web Server

Nginx

### 您正在使用宝塔吗？

- [X] 是

### 出现了什么问题？

**登陆 注册 密码重置页面** 的输入框全都加载不出来
![StarSkin](https://pic-up.star-skin.cn/i/2023/05/02/7b1f6d1d-a93f-0fed-5f41-49ba7f3e048d.png)

### 错误日志

```text
无日志
```

### 重现步骤

一直刷新，或清理缓存

**Comment by skyfsj** [comment]
检查站点地址是否填写正确
